trunk/NDHMS/MPP/mpp_land.F

```diff
@@ -152,8 +152,8 @@ subroutine MPP_LAND_INIT()
       CALL mpi_initialized( mpi_inited, ierr )
       if ( .NOT. mpi_inited ) then
            call MPI_INIT( ierr )  ! stand alone land model.
-      endif
            HYDRO_COMM_WORLD = MPI_COMM_WORLD
+      endif
```

Comment:
I moved HYDRO_COMM_WORLD = MPI_COMM_WORLD into the block of code that initializes MPI because concurrently running coupled models will run on different sub communicators. If mpi has been initialized in a external driver, such as a NUOPC driver, then HYDRO_COMM_WORLD should not be overwritten with MPI_COMM_WORLD. There is a similar block of code in MPP/CPL_WRF.F:71.

In the coupled NUOPC use case the NUOPC cap will directly set HYDRO_COMM_WORLD using a call to ESMF_VMGet at the beginning of the ini phase (see CPL/NUOPC_cpl/WRFHydro_NUOPC_Gluecode.F90:305)

Reply:
👍

Reply:
@jmccreight It looks good to me.